Documentation Directory/Cavas_api_documentation.md

```diff
@@ -0,0 +1,441 @@
+﻿# Canvas LMS API
+
+**API Base URL**: `url = "https://canvas.example.com/api/v1/"`
+
+This is where the specific endpoints must be concatenated at.
+
+## Authentication
+
+Ensure that an API Token has been obtained before sending requests, and define them as such: `headers = {'Authorization': 'Bearer [api_token_here]'}`
+
+## Sending requests
+
+Using the requests dependency to fetch data as a JSON object: `requests.get(url, headers=headers)`
```

Comment:
You could capitalize the name Requests dependency. That's how they reference themselves [Requests website](https://docs.python-requests.org/en/latest/index.html)